connector/ldap/ldap.go

```diff
@@ -425,6 +445,7 @@ func (c *ldapConnector) Login(ctx context.Context, s connector.Scopes, username,
 		if ident.ConnectorData, err = json.Marshal(refresh); err != nil {
 			return connector.Identity{}, false, fmt.Errorf("ldap: marshal entry: %v", err)
 		}
+		c.logger.Debugln("ldap: configured offline access")
```

Comment:
The placement of this message is not really correct because this is not where the user sets the offline access. The order can be confusing. Please remove